src/components/filter/filter.tsx

```diff
@@ -14,6 +14,14 @@ export const Filter: React.FC = () => {
 	const { isLoggedIn } = useAuthStore();
 	const [isTooltipVisible, setIsTooltipVisible] = useState(false);
 
+	useEffect(() => {
+		if (isTooltipVisible) {
+			setTimeout(() => {
+				setIsTooltipVisible(false);
+			}, 5000);
+		}
+	}, [isTooltipVisible]);
```

Comment:
I think we should rather use an early exit (to prevent the multi-indent nesting) and we should use a cleanup function in case this gets triggered multiple times in a row.

```typescript
useEffect(() => {
  if (!isTooltipVisible) {
    return () => {}
  }

  const timeoutId =  setTimeout(() => {
    setIsTooltipVisible(false);
  }, 5000);

  return () => clearTimeout(timeoutId);
}, [isTooltipVisible]);
```